packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -85,5 +85,10 @@ export const messages = {
   force_config_list_text: 'SFDX: List All Config Variables',
   force_alias_list_text: 'SFDX: List All Aliases',
   force_org_display_default_text:
-    'SFDX: Display Org Details for Default Scratch Org'
+    'SFDX: Display Org Details for Default Scratch Org',
+
+  force_data_soql_query_input_text: 'SFDX: Execute SOQL Query',
```

Comment:
Because additional input is needed, this should end in an ellipsis:
'SFDX: Execute SOQL Query...'